Bottom: 1ae738a42c25e6d2f1c92e4e79e5653b620de2c4
Top:    0e6b8c5a5f999b6695127164e8c2493cb15abf49
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-25 17:12:24 -0700

Refresh of shut_wr

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 63bf03e..0bba7e5 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -96,8 +96,7 @@ enum {
 #define rs_msg_data(imm_data) (imm_data & 0x1FFFFFFF)
 
 enum {
-	RS_CTRL_DISCONNECT,
-	RS_CTRL_SHUTDOWN
+	RS_CTRL_DISCONNECT
 };
 
 struct rs_msg {
@@ -884,8 +883,6 @@ static int rs_poll_cq(struct rsocket *rs)
 				if (rs_msg_data(imm_data) == RS_CTRL_DISCONNECT) {
 					rs->state = rs_disconnected;
 					return ERR(ECONNRESET);
-				} else if (rs_msg_data(imm_data) == RS_CTRL_SHUTDOWN) {
-					rs->state = rs_shutdown_rd;
 				}
 				break;
 			default:
